### DISCUSSION TITLE
getaddrinfo EAI_AGAIN localhost

### DISCUSSION THREAD

**Discussion by mrScalder** [outline/outline#3896] [Self Hosting] [answered]
Hi,

I try to setup outline server on vds with command:

`sudo bash -c "$(wget -qO- https://raw.githubusercontent.com/Jigsaw-Code/outline-server/master/src/server_manager/install_scripts/install_server.sh)"`

Installation stack on line:
```
> Removing watchtower container ............... OK
> Restarting watchtower ....................... OK
> Waiting for Outline server to be healthy .... 
```

On comand `docker logs shadowbox` I see these log:
```
D2022-07-30T07:27:35.506Z 12 main.js:29] SB_METRICS_URL: https://prod.metrics.getoutline.org
D2022-07-30T07:27:35.506Z 12 main.js:29] ==============
I2022-07-30T07:27:35.506Z 12 main.js:29] Starting...
E2022-07-30T07:27:40.517Z 12 main.js:29] Error: getaddrinfo EAI_AGAIN localhost
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:64:26)
```

Please, help to find out how to fix this issue or where I can find help. Thanks.

**Comment by tommoor** [comment, marked as answer]
Wrong project: https://github.com/Jigsaw-Code
